commands/add_service.go

```diff
@@ -0,0 +1,25 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/cristianoliveira/ergo/proxy"
+)
+
+// AddService Allows to add new services to existing configuration file
+// Usage:
+// `ergo add service servicehost:port`
+func AddService(config *proxy.Config, service proxy.Service, configFile string) {
+	old_service := config.GetService(service.Name + config.Domain)
```

Comment:
@sitaramshelke - rename `old_service` to `oldService` for `golint`. Can you additionally run `golint` for `add_service.go`?

Reply:
@srini156 Good find. I actually I had setup VSCode to use golint. Will check the settings as well.
Thanks.